custom_model_runner/datarobot_drum/drum/perf_testing.py

```diff
@@ -682,9 +683,13 @@ def check_transform_server(self, target_temp_location=None):
 
             response = requests.post(run.url_server_address + endpoint, files=payload)
             if not response.ok:
-                raise DrumCommonException(
-                    "Failure in {} server: {}".format(endpoint[1:-1], response.text)
-                )
+                error_msg = "Failure in {} server: {}".format(endpoint[1:-1], response.text)
```

Comment:
what is `endpoint[1:-1]`? `transform/` ? Lets add that in the error message without formatting

Reply:
We should probably have an equivalent message for predict servers

Reply:
With the existing implementation, output schema validation is only available for transformer. When output schema shows up in model-metadata of an estimator, it raises error before fitting.
https://github.com/datarobot/datarobot-user-models/blob/b5c58066edda6d9698836f92ee30864110ae2c04/custom_model_runner/datarobot_drum/drum/drum.py#L502 

With that, the output schema validation and follow-up exception handling (this pr) are only available in transform case.